blog/_posts/btcpay-server-1-10-0.md

```diff
@@ -0,0 +1,119 @@
+
+## 🧾New Form Builder Drag and Drop UI!
+
+Ever since we’ve [introduced](https://blog.btcpayserver.org/btcpay-server-1-8-0/) custom forms, which allow you to collect various customer information during the checkout so you can process orders in a more personalized manner, we’ve been hearing your feedback around the complexity of using them. 
+
+In this release, we’re introducing a drag-and-drop form builder UI, enabling you to design customizable forms to your liking.
+
+Why tell you about the drag-and-drop form builder when we can show it? 
+In this custom form, let’s use the restaurant use case as an example again. In-Store settings -> Forms -> Click on “Create a new form.” When creating a new form, we always start with a default blank “element”. Set the field values on the right side of the form builder. Once you’ve set it, we can click “Add new element” at the bottom of your first, for the second field, and so on. 
+
+![BTCPay Server custom forms example](/images/BTCPayServerForms.gif)
+
+Did you get to play around with forms already? Please share your experience with us on Twitter, Nostr or come join our Mattermost chat! We’d love to hear from you! For more details on form builder, check our [documentation](https://docs.btcpayserver.org/Forms/).
+
+## 🔒Custom role system 
+
+Are you currently operating a store with a team of employees? If the existing guest and admin roles haven't quite met your requirements, we have some exciting news for you. 1.1.10 ships with a brand-new Custom Role management system.
+
+This upgrade empowers you to tailor roles to your exact needs, granting your staff access to specific permissions within your store. Whether you need roles for cashiers, administrative personnel, or order pickers in your warehouse, you can now effortlessly create as many custom roles as necessary, streamlining and optimising your store's operations.
+
+![BTCPay Server custom roles](/images/BTCPayServerRoles.gif)
+
+## 🥷 Hide sensitive information
+
+If you ever opened your BTCPay in a coffee shop or a coworking space we’re sure you were nervous to not accidentally reveal your wallet balance or invoice amounts. The new privacy-enhancing feature within your Account Settings will allow you to conveniently hide such information from prying eyes. 
+
+To try it out, click on your Account in the sidebar’s bottom left > Toggle Hide Sensitive information to ON, all value fields will be hidden. From Dashboard to every field in an invoice, none are displayed with value; all are masked.
+
+![BTCPay Server Hide sensitive info](/images/BTCPayServerHiden.gif)
```

Comment:
This is the LNbank GIF. We should create a separate one for the Hide Sensitive Information feature, this one isn't clear. Let's move this one to the LNbank section.